### PR TITLE
docs: Toolbox - add example image of schema view

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

Updating the Docs for the GraphQL Toolbox to include more visuals.

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Low

### CODE REVIEW

File: docs/modules/ROOT/pages/toolbox.adoc [hunk 1]
```diff
@@ -4,24 +4,34 @@
...
 
 == Requirements
 
 1. A Neo4j database, see https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or use a https://neo4j.com/cloud/[Neo4j AuraDB]
-2. Make sure the database fulfills the requirements stated xref::introduction.adoc#introduction-requirements[here], including the necessary plugins.
+2. Make sure the Neo4j database fulfills xref::introduction.adoc#introduction-requirements[these requirements], including the necessary plugins
+
+== Example
+
+Define your Type definitions directly in the editor or Introspect the Neo4j database you connected to.
```
Comment (mjfwebb) [inline docs/modules/ROOT/pages/toolbox.adoc:27]: Is it "Type definitions", or "type definitions"?

Comment (mjfwebb) [reply docs/modules/ROOT/pages/toolbox.adoc:27]: also "Introspect" or "introspect"?

If it's a verb, I would suggest the lowercase variant.

File: docs/modules/ROOT/pages/toolbox.adoc [hunk 2]
```diff
@@ -4,24 +4,34 @@
...
+
+image::toolbox-schema-view-v2.png[title="Type definitions view"]
+
+Then query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library.
+
+image::toolbox-editor-view-v2.png[title="Query editor view"]
+
+Note that you can store your Type definitions as favorites and that a range of settings are available to work with.
```
Comment (mjfwebb) [inline docs/modules/ROOT/pages/toolbox.adoc:35]: Same question here about "Type" versus "type"

**Review by mjfwebb — COMMENTED** [review]
Just a few questions

File: docs/modules/ROOT/pages/toolbox.adoc (hunk 1 above)
Comment (tbwiss) [reply docs/modules/ROOT/pages/toolbox.adoc]: https://github.com/neo4j/graphql/pull/3508/commits/fe971fe9f08ed898b186185ee1167ac0c837ad9a

File: docs/modules/ROOT/pages/toolbox.adoc (hunk 2 above)
Comment (tbwiss) [reply docs/modules/ROOT/pages/toolbox.adoc]: https://github.com/neo4j/graphql/pull/3508/commits/fe971fe9f08ed898b186185ee1167ac0c837ad9a

**Review by mjfwebb — APPROVED** [review]
Thanks!

**Pull request merged**